scripts/check_imports_and_dependencies.py

```diff
@@ -60,6 +58,10 @@ class DependenciesTool:
     @staticmethod
     def get_package_files(package_name: str) -> List[Path]:
         """Get package files list."""
+        from pip._internal.commands.show import (  # type: ignore  # noqa  # pylint: disable=import-outside-toplevel
+            search_packages_info,
+        )
+
```

Comment:
Why moved here?

Reply:
did a small fix mfor this script, was a conflict with some import order